.github/workflows/ci.yaml

```diff
@@ -0,0 +1,24 @@
+on: push
+jobs:
+  test:
+    strategy:
+      matrix:
+        go: [ '1.10.x', '1.11.x', '1.12.x' ]
```

Comment:
also add `1.13.x`

Reply:
We can't support for 1.13.x and I don't understand fix.
https://github.com/swaggo/swag/pull/507